cmd/helm/release_testing_run.go

```diff
@@ -44,27 +42,7 @@ func newReleaseTestRunCmd(cfg *action.Configuration, out io.Writer) *cobra.Comma
 		Long:  releaseTestRunHelp,
 		Args:  require.ExactArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			c, errc := client.Run(args[0])
-			testErr := &testErr{}
-
-			for {
-				select {
-				case err := <-errc:
-					if err != nil && testErr.failed > 0 {
-						return testErr.Error()
-					}
-					return err
-				case res, ok := <-c:
-					if !ok {
-						break
-					}
-
-					if res.Status == release.TestRunFailure {
-						testErr.failed++
-					}
-					fmt.Fprintf(out, res.Msg+"\n")
-				}
-			}
+			return client.Run(args[0])
```

Comment:
Looks like we're not supporting running tests in parallel at the moment in helm 3 so I like this simplification.